src/foam/u2/view/OverlayActionListView.js

```diff
@@ -7,18 +7,21 @@
 foam.CLASS({
   package: 'foam.u2.view',
   name: 'OverlayActionListView',
-  extends: 'foam.u2.View',
+  extends: 'foam.u2.tag.Button',
```

Comment:
Another approach could be to have a generic dropdownButtonView that this view extends as this one is restricted to adding only actions to the overlay but that could be done once we use the overlay/dropdown button for more things